scripts/rules.matterlint

```diff
@@ -79,9 +79,25 @@ all endpoints {
 
 endpoint 0 {
   // Identifiers for clusters are loaded from XML files
+  // The required clusters are from the spec for RootNode (section 2.1.5 at the moment):
+
   require server cluster Basic;
+  require server cluster AccessControl;
   require server cluster GroupKeyManagement;
-  require server cluster NetworkCommissioning;
   require server cluster GeneralCommissioning;
-  require server cluster OTASoftwareUpdateRequestor;
+  require server cluster AdministratorCommissioning;
+  require server cluster OperationalCredentials;
+  require server cluster GeneralDiagnostics;
```

Comment:
Using these names, rather than IDs, means that if we rename anything, we have to change the rules.

Suggest being able to list clusters by ID.

Reply:
We use xml inside our matter repository right? So renaming should be in sync.

I can add by ID easy enough, however that is not friednly for humans. What is `require server cluster 0x003E`? I can add a comment of `// Operational Credentials` but then I rely that the comment matches the code.

Reply:
created https://github.com/project-chip/connectedhomeip/pull/19187 to support numbers if we chose so. I believe the more likely scenario would be validating without XML (for whatever reason). For now I believe linters should use constants as they are human friendly